### PR TITLE
Update LEMAscent.cfg for 1.10.1

### PR DESCRIPTION
**Pull request by Jeceratops**
Patch is originally made by Pap and makes the LEM work on 1.10.1 as intended. Doesn't freeze on loading. Only affects the LEMs config, Untested on 1.8.1

Thanks Standecco!

### CODE REVIEW

File: GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg [hunk 1]
```diff
@@ -1,133 +1,347 @@
...
 	module = Part
+	name = ROC-LEMAscent
 	author = DECQ, Pap
-
+	
 	RSSROConfig = true
-
-	//	============================================================================
+	RP0conf = true
+
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg:9]: Do not set RP0conf here.  That is RP0's job.

File: GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg [hunk 2]
```diff
@@ -1,133 +1,347 @@
...
+        rotation = 0.0, 0.0, 0.0
+    }
+	
 	scale = 1.0
 	rescaleFactor = 1.0
-
+	
 	node_stack_top = 0.0, 1.405, 0.0, 0.0, 1.0, 0.0, 1
-	node_stack_bottom = 0.0, -1.375, 0.0, 0.0, -1.0, 0.0, 2
+    node_stack_bottom = 0.0, -1.375, 0.0, 0.0, -1.0, 0.0, 2
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg:34]: Why all the messing with whitespace, to make things -not- consistent?

File: GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg [hunk 3]
```diff
@@ -1,133 +1,347 @@
...
-	manufacturer = Grumman Aerospace Corporation
-	description = The ascent stage and crew compartment of the Apollo Lunar Module. This has the ascent engine integrated.
+	
 	tags = apollo, moon, lunar, crew, armstrong, aldrin, cm, csm, saturn, v, lem, lm, module, ascent
-
+	
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg:61]: Why all the re-ordering?  Makes it look like a lot more changes that someone needs to review and see that, in fact, nothing changed.

File: GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg [hunk 4]
```diff
@@ -1,133 +1,347 @@
...
+	//  ============================================================================
+	
 	MODULE
 	{
 		name = ModuleCommand
 		minimumCrew = 1
-		hasHibernation = True
-		hibernateOnWarp = True
-		hibernationMultiplier = 0.65
+        hasHibernation = True
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg:102]: Stop breaking the indenting ;)

File: GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg [hunk 5]
```diff
@@ -363,463 +581,235 @@ PART
...
 }
 
-//	================================================================================
-//	Add RasterPropMonitorComputer Module
-//	================================================================================
+//  ================================================================================
+//	Configure RemoteTech
+//  ================================================================================
 
-@PART[ROC-LEMAscent]:NEEDS[RasterPropMonitor]
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/LEM/LEMAscent.cfg:503]: Why did you break this patch structure, and not include NEEDS[RasterPropMonitor] when you declared its fields above instead?

**Review by DRVeyl — CHANGES_REQUESTED** [review]
Can you fix some of the issues, and possibly remove a lot of the "changes" that actually make all the whitespace/indentation actively worse?

**Comment by Shaggygoblin** [comment]
so... this does work to fix the 1.10.x load freeze, arguable whitepace concerns aside?

**Comment by DRVeyl** [comment]
Only laughable if you have no role in maintaining the codebase or in QC'ing changes across several projects.
Also was reported that this patch breaks animations.  So feedback on that would be appreciated.
You can view the file directly here by going to the FIles Changed page, then selecting View File in:
image

and pasting the result into your own copy of the original.  You can also visit Jceratops' fork and get the file directly, rather than cloning/forking his repo.

**Comment by Shaggygoblin** [comment]
This does, indeed, fix the NRE Freeze while loading LEM for me. Usability confirmed, if not readability.

aside: While I do understand and share your view on readability, I find it trivial in weight against functionality.  When peer'ing, I look past the readability flaws and get to the meat of it: does it work or not?  If it works, we clean it up.  If not, fix it. Rinse, repeat.

**Comment by vader111** [comment]
Just me looking at how the code has been changed is making me need to bleach my eyes out, your basically undoing all the formatting changes i made to tidy it up and standardize it.

**Comment by vader111** [comment]
@Shaggygoblin all you needed to do was add useMultipleDragCubes = false to all the ModuleAnimateGeneric modules.

**Comment by vader111** [comment]
@Shaggygoblin Might be easier for you to close this PR and let me create a PR.

**Comment by Shaggygoblin** [comment]
I didn't modify anything.  My musings were on the content as it was.  I merely added this file to the install, and it worked to finish loading.  I was asked to feedback on animations and I did.  I made no modifications to the code.

**Pull request closed without merging**